src/Workspaces/CSharp/Portable/CodeGeneration/ConstructorGenerator.cs

```diff
@@ -119,7 +120,7 @@ private static ArgumentListSyntax GenerateArgumentList(ImmutableArray<SyntaxNode
 
         private static SyntaxTokenList GenerateModifiers(IMethodSymbol constructor, CodeGenerationOptions options)
         {
-            var tokens = new List<SyntaxToken>();
+            var tokens = ArrayBuilder<SyntaxToken>.GetInstance();
```

Comment:
does the call to 'ToSyntaxTokenList' below return the ArrayBuilder back to the pool?

Reply:
use ToSyntaxTokenListAndFree below.

Reply:
Done